src/storage/v2/inmemory/storage.cpp

```diff
@@ -670,25 +670,19 @@ utils::BasicResult<StorageManipulationError, void> InMemoryStorage::InMemoryAcce
     // Replica can log only the write transaction received from Main
```

Comment:
How are we sure that committed transactions are sorted by the commit timestamp in the WAL files